src/main/python/org/broadinstitute/hellbender/gcnvkernel/models/model_ploidy.py

```diff
@@ -252,7 +252,10 @@ def __init__(self,
                       * ploidy_k.dimshuffle('x', 'x', 0))
         mu_den_sjk = gamma_rest_sj.dimshuffle(0, 1, 'x') + mu_num_sjk
         eps_mapping_j = eps_mapping * t_j / tt.sum(t_j)  # average number of reads erroneously mapped to contig j
-        mu_sjk = ((1.0 - eps_mapping) * (mu_num_sjk / mu_den_sjk)
+
+        # the switch is required for a single contig edge case
+        mu_ratio_sjk = tt.switch(tt.eq(mu_den_sjk, 0.0), 0.0, mu_num_sjk / mu_den_sjk)
```

Comment:
Actually, despite our back and forth on Slack, I'm not convinced there's a good argument either way for

`tt.switch(tt.eq(mu_den_sjk, 0.0), 0.0, mu_num_sjk / mu_den_sjk)`
vs.
`tt.switch(tt.eq(mu_den_sjk, 0.0), 1.0, mu_num_sjk / mu_den_sjk)`.

In the first, we heavily penalize the k = 0 case by saying the Poisson rate is eps_mapping * n_s, whereas the likelihood is flat and the Poisson rate is n_s for all k > 0.  In the second, the likelihood is flat for all k.

In either case, in practice (where k = 0 is already heavily penalized by the prior), inference is completely determined by the prior, since mean bias and ploidy cancel out of the likelihood.